spec/models/visit_spec.rb

```diff
@@ -53,16 +53,26 @@
       .to raise_error ActiveRecord::RecordInvalid
   end
 
-  it "is invalid with a start date in the past" do
+  it "is invalid with a start date two days ago" do
     expect do
       FactoryGirl.create(
         :visit,
-        start_date: Faker::Date.backward(1),
+        start_date: Date.current - 2.day,
```

Comment:
Faker::Date was causing my test to flap.  backward(2) gives a random date in the past no more than 2 days back but we needed one at least 2 days back.  Not a very clear method name.